packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -121,22 +131,221 @@ export default class VTable {
             return;

```

Comment:
When using adding columns and writeback, on a table that I applied styles, the header row height is decreased and the styles removed.
I think we should call the applyformat inside of the writeback if the table have tableInfo
![TABL1](https://user-images.githubusercontent.com/8291124/149350327-232b4d55-1e11-48fd-ba28-07eb5d950665.gif)
o